src/components/AppSettingsMenu.vue

```diff
@@ -18,7 +18,11 @@
 			/>
 			<label for="gravatar-enabled">{{ t('mail', 'Use Gravatar and favicon avatars') }}</label>
 		</p>
-
+		<p>
+			<button class="icon-mail-white app-settings-button primary" @click="registerProtocolHandler">
```

Comment:
let's use the non-white icon and remove `primary`. This is a secondary action :)